ee/tabby-ui/components/chat.tsx

```diff
@@ -122,12 +152,17 @@ function ChatRenderer(
   }, [messages])
 
   React.useEffect(() => {
-    const scrollHeight = document.documentElement.scrollHeight
-    window.scrollTo(0, scrollHeight)
+    scrollToBottom()
 
     return () => stop()
   }, [])
 
+  React.useLayoutEffect(() => {
+    if (isStreamResponsePending) {
+      scrollToBottom('smooth')
```

Comment:
call to `scrollToBottom` should be debounced (e.g every 5 seconds) to make the UX more friendly

Reply:
This callback will only be executed once after appending a request message

Reply:
but appending message could happens at higher frequency (e.g much faster than every 5s)